sdk/core/azure-json-reflect/pom.xml

```diff
@@ -50,6 +50,8 @@
 
     <javadoc.excludePackageNames/>
     <javadoc.sourcepath>${basedir}/src/main/java</javadoc.sourcepath>
+
+    <checkstyle.suppressionsLocation>checkstyle-suppressions.xml</checkstyle.suppressionsLocation>
   </properties>
```

Comment:
Same question about having no SpotBugs file for this package.

Reply:
This doesn't ship so I left Spotbugs disabled for now